cyvcf2/cyvcf2.pyx

```diff
@@ -950,7 +954,7 @@ cdef class Variant(object):
             cdef int unknown = 3 if self.vcf.gts012 else 2
             for i in range(0, n * self.vcf.n_samples, n):
                 if n == 2:
-                    if gt_types[j] == unknown:
+                    if (gt_types[j] == unknown) and (self.vcf.missingness == False):
```

Comment:
Yeah.  I don't like the term `missingness` either, but I've still yet to think of a better term for this feature.